service/service.go

```diff
@@ -435,7 +447,12 @@ func createURLBuilder(cfg *config.Configuration) (*url.Builder, error) {
 		return nil, errors.Wrap(err, "unable to parse importAPIURL from config")
 	}
 
-	return url.NewBuilder(websiteURL, downloadServiceURL, datasetAPIURL, codeListAPIURL, importAPIURL), nil
+	APIRouterPublicURL, err := neturl.Parse(cfg.APIRouterPublicURL)
```

Comment:
Is there any reason why this variable has been name starting in uppercase when all the others begin with lowercase letters?